nipap/nipap/smart_parsing.py

```diff
@@ -429,8 +433,6 @@ def _string_to_dictsql(self, part):
                     'val2': part[0]['word']
                     }
 
-        elif part.getName() == 'ipv4_address':
-            dictsql = {}
```

Comment:
I'm not quite sure how the other changes introduced relates to this change - was this never used?